### PR TITLE
Fix docs for verifier

### PR DESCRIPTION
**Pull request by drudenko**
@electrum please review

### CODE REVIEW

**Comment by dain** [comment]
looks good

File: presto-docs/src/main/sphinx/installation/verifier.rst [hunk 1]
```diff
@@ -16,12 +16,16 @@ Create a MySQL database with the following table and load it with the queries yo
         name VARCHAR(256),
         test_catalog VARCHAR(256) NOT NULL,
         test_schema VARCHAR(256) NOT NULL,
+        test_prequeries TEXT NULL,
```
Comment (electrum) [inline presto-docs/src/main/sphinx/installation/verifier.rst]: Don't add "null" since that is the default.

**Comment by electrum** [comment]
Use the imperative mood for commit messages: http://chris.beams.io/posts/git-commit/

So it should be "fix" not "fixed"

File: presto-docs/src/main/sphinx/installation/verifier.rst (hunk 1 above)
Comment (drudenko) [inline presto-docs/src/main/sphinx/installation/verifier.rst]: ok

**Comment by drudenko** [comment]
ok

**Comment by drudenko** [comment]
ping

**Comment by martint** [comment]
Merged, thanks!

**Pull request closed without merging**